service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/service/FeeDecisionGeneratorIntegrationTest.kt

```diff
@@ -3182,14 +3182,11 @@ class FeeDecisionGeneratorIntegrationTest : FullApplicationTest(resetDbBeforeEac
     }
 
     @Test
-    fun `a new fee decision is not generated if incomes change from NOT_AVAILABLE to INCOMPLETE`() {
+    fun `before 03-25 a new fee decision is not generated if income changes from NOT_AVAILABLE to INCOMPLETE`() {
```

Comment:
Ehkä käyttäisin muotoa `3/2025` (suomi) tai `2025-03` (iso), itselle in ainakin vaikea ymmärtää mitä `03-25` tarkoittaa.